types/wordpress__components/card/body/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import Card from '..';
+import { StyledComponent } from '@emotion/styled';
+
+declare namespace CardFooter {
+    interface Props {
+        /**
+         * `className` of the container.
+         */
+        className?: string;
+
+        /**
+         * Add visual shading style.
+         */
+        isShady?: boolean;
+
+        size?: Card.CardSize;
+    }
+}
+
+declare const CardFooter: StyledComponent<JSX.IntrinsicElements['div'], CardFooter.Props, {}>;
```

Comment:
When did wordpress start using emotion?

Reply:
Only [recently](https://github.com/WordPress/gutenberg/pull/17963) -- `Card` is the first component to use it AFAIK.